### DISCUSSION TITLE
ユーザーページの「LGTMした記事についていたタグ上位5つ」を非表示にしたい

### DISCUSSION THREAD

**Discussion by k1350** [increments/qiita-discussions#88] [要望・アイデア・不具合]
ユーザーページ右上の「$ analyze [ユーザーID]」のところにある「LGTMした記事」を非表示にしたいです。

理由について、私の具体例をあげさせていただきます。

私は Qiita アドベントカレンダーの時期に Qiita を使い始め、とあるカレンダーを購読し、記事を幾つかLGTM（実際にLGTMした当時の名称は「いいね」）していました。
そのカレンダーの投稿記事には「クソアプリ」というタグがついていることが多く、一方で使用言語は記事ごとにバラバラでした。
そのためユーザーページリニューアルされたとき、私のユーザーページに表示された「LGTMした記事」の集計結果は、第一位が「クソアプリ」になっていました。

[ユーザーページリニューアルの際のブログ](https://blog.qiita.com/user-page-renewal/)では

> 今回のリニューアルでは、より「どんなエンジニアなのか」を自分自身で見ても気づきを得られ、他の人が見れば自分がどんなエンジニアなのかがわかるページを目指しました。

と説明されています。

私は投稿記事の技術的な部分についていいと思ったのであって、別に「クソアプリ」に興味があったわけではありません。
たまたま記事にそのようなタグがついていただけです。
それなのにあたかも「クソアプリ」に興味があるかのようになってしまいました。
タグ情報だけを集計して目立つように表示することは、「どんなエンジニアなのか」の正確な表現にならないばかりか、イメージに悪影響を及ぼすことすらあり得ます。

結局、私はユーザーページリニューアル直後に過去のすべてのLGTMを取り消しました。
アカウントが所属企業に紐づいていたので、あまりきれいでない言葉がユーザーページに表示されるのを避けたかったという理由もあります。

その後、現在に至るまで、LGTMは押さないようにしています。

その記事についているタグがユーザーページに目立つように表示されることが自分にとってプラスになるかどうか、などということを考えてLGTMするかどうか決めたくありません。
単に内容がいいと思った記事にLGTMしたいです。

以上の理由から、LGTMした記事についていたタグ上位5つを非表示にしたいです。
本件はユーザーページリニューアル当時に意見として送りましたが、再度意見表明させていただきます。

**Comment by kskwtnk** [comment]
@k1350 

ご意見ありがとうございます！

こちらについては以前意見をいただいており、検討には上がったことはあるものの引き続き表示をしていました

恐らくk1350さんの意見も当時リストアップさせていただいていたと思います
再度意見を表明してくださりありがとうございます！

こちら改めて社内で議論を実施します！
すぐにどうすると確約できず申し訳ありませんが今しばらくお待ちいただけると幸いです 🙇

**Comment by kskwtnk** [comment]
@k1350 
お返事が遅くなってしまいすみません！社内で議論を実施して参りました 🙏 

結論として「単にLGTMの情報を消すというより、そもそもユーザーページとしてもっと役立つ情報設計にアップデートしていこう」といったニュアンスで着地しております

なお、確定ではありませんが、今のように全員一律でLGTMしたタグが出ることはなくなっていきそうです
（詳細はまだこれからも議論があるので、ひとまずの情報として捉えていただけると幸いです 🙇 ）

マイページ自体のアップデートについて続報があればDiscussionsでもお知らせしようと思いますし、ベータ版での提供などもいたします！

要望をいただいた内容そのものへのお返事ではありませんが、根本に立ち返って議論することができました、ありがとうございます！

**Reply by k1350** [reply]
@xrxoxcxox ありがとうございます。
> ユーザーページとしてもっと役立つ情報設計にアップデートしていこう

というのが最終的に私にとっても望ましいユーザーページになるかはわかりませんが、また気兼ねなく他の方の記事を評価できるようになれば嬉しいです。

[コミュニティガイドライン](https://help.qiita.com/ja/articles/qiita-community-guideline) のページにある
> また読んで面白いと思った記事を評価することを通して、あなたがどんな強みや関心を持ったエンジニアなのか表現することをQiitaは支援します。

という部分がおそらく私の感覚とQiitaというプラットフォームの思想のズレで、私は自己表現のために他の方の記事を評価したいわけではありません。
つまりたとえば「Javaに興味があるエンジニアだと思われたいからJavaの記事にLGTMしよう」などとは一切思ったことがありません。
ただその記事を読んで助かったとか、面白いと思ったとか、そういうことを筆者に伝えたいだけです。

同ガイドラインには
> 読んでみて「この記事が役に立った！」と思ったら、LGTMやコメントなどでぜひその気持を伝えてください。

とも書いてあるので、Qiitaの思想としてLGTMとはいったい何のための物なのかは正確にはわからないのですが、自己表現のためにLGTM情報を使いたくないユーザーも受け入れてもらえるプラットフォームならありがたく思います。

**Comment by kyntk** [comment]
本日[LGTMをいいねに変更しました](https://github.com/increments/qiita-discussions/discussions/283)が、同時に「LGTMした記事」を表示しないようにしました!

大変お待たせいたしました。 🙇
